order.go

```diff
@@ -46,9 +52,11 @@ type OrderReturnParams struct {
 }
 
 type Shipping struct {
-	Address Address `json:"address"`
-	Name    string  `json:"name"`
-	Phone   string  `json:"phone"`
+	Address        Address `json:"address"`
+	Name           string  `json:"name"`
+	Phone          string  `json:"phone"`
+	Carrier        string  `json:"carrier"`
+	TrackingNumber string  `json:"tracking_number"`
```

Comment:
And same here — just move `Carrier` into the right place.

Reply:
Done! Can't believe I missed that since I've left that same comment on countless PRs... :)

Reply:
Haha! Thanks again.